### PR TITLE
Release/1.12.0

### PR DESCRIPTION
**Pull request by Steffengreiner**
This release introduces an abstract builder pattern for the business DTOs of the QBiC business domain and adds support for the QBiC Template Sync and Github Actions

### CODE REVIEW

File: qube.cfg [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 1.12.0-SNAPSHOT
+current_version = 1.12.0
```
Comment (jenniferboedker) [inline qube.cfg:2]: did you use qube to bump the version?

**Review by jenniferboedker — COMMENTED** [review]
So depending on if you used qube for the bump of the version I would either approve or request changes ;)

File: qube.cfg (hunk 1 above)
Comment (Steffengreiner) [reply qube.cfg:2]: Yes i used the qube `bump-version` command with the qube 2.5.1 version to increase the versioning of `data-model-lib`
Interesting that it skipped to do so in the `qube.cfg `Nice Catch 👍

Comment (jenniferboedker) [reply qube.cfg:2]: No its fine there is no problem. I was just curious in general.

**Review by jenniferboedker — APPROVED** [review]
LGTM

**Pull request merged**